donate/settings/testing.py

```diff
@@ -9,7 +9,7 @@
 
```

Comment:
Added this var, so we do not send emails while testing

Reply:
See below comment. Good for now, though we'll need to find a better solution in the future.